apps/desktop/src/renderer/renderer.js

```
@@ -0,0 +1,79 @@
+  try {
+    await window.desktop.retrySelection();
+  } catch (e) {
+    showError(e.message);
+  }
+};
+$("close").onclick = () => window.desktop.close();
+$("send").onclick = async () => {
+  try {
+    showError();
+    answer.textContent = "";
+    await window.desktop.ask(selection.value, question.value);
```

ESLint

[error] 'window' is not defined.

(no-undef)